opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/netflow5.adoc

```diff
@@ -27,6 +27,8 @@ The Netflow v5 UDP Parser supports protocol detection.
 | `maxClockSkew`        | The maximum delta in seconds between exporter and Minion timestamps.       | no       | 0
 | `clockSkewEventRate`  | Used to rate-limit clock skew events in seconds.                           | no       | 3600
 | `dnsLookupsEnabled`      | Used to enable or disable DNS resolution for flows.                        | no       | true
+| `flowActiveTimeoutFallback`   | Fallback value for active flow timeout if setting value is not included in exported flows | no | null
+| `flowInactiveTimeoutFallback`   | Fallback value for inactive flow timeout if setting value is not included in exported flows | no | null
```

Comment:
These values are defined in the documentation but are not implemented in the parser.

Reply:
oh, good catch. This was already an oversight in the already merged version. Will fix.